src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        super.draw();
```

Comment:
Ensure that the `super.draw()` call is necessary. This depends on the implementation of the `Figure` class. If `Figure` does not have a `draw()` method or if it does not need to be called, this line can be removed.